src/generated/design-tokens-base.css

```diff
@@ -93,23 +103,24 @@
   --echoes-text-case-none: none;
   --echoes-text-decoration-none: none;
   --echoes-text-decoration-underline: underline;
-  --echoes-typography-paragraph-default-regular: 400 0.875rem/1.42 Inter;
-  --echoes-typography-paragraph-default-medium: 500 0.875rem/1.42 Inter;
-  --echoes-typography-paragraph-default-semi-bold: 600 0.875rem/1.42 Inter;
-  --echoes-typography-paragraph-small-regular: 400 0.75rem/1.42 Inter;
-  --echoes-typography-paragraph-small-semi-bold: 600 0.75rem/1.42 Inter;
-  --echoes-typography-paragraph-large-regular: 400 1rem/1.42 Inter;
-  --echoes-typography-paragraph-large-semi-bold: 600 1rem/1.42 Inter;
-  --echoes-typography-heading-small: 600 1rem/1.32 Inter;
-  --echoes-typography-heading-medium: 600 1.25rem/1.32 Inter;
-  --echoes-typography-heading-large: 600 1.5rem/1.32 Inter;
-  --echoes-typography-heading-xlarge: 600 2.125rem/1.32 Inter;
-  --echoes-typography-code-default: 400 0.875rem/1.42 'Ubuntu Mono';
-  --echoes-typography-code-highlight: 700 0.875rem/1.42 'Ubuntu Mono';
-  --echoes-typography-code-comment: 400 italic 0.875rem/1.42 'Ubuntu Mono';
-  --echoes-typography-others-label: 600 0.875rem/1.42 Inter;
-  --echoes-typography-others-link: 600 0.875rem/1.42 Inter;
-  --echoes-typography-others-link-large: 600 1rem/1.42 Inter;
+  --echoes-typography-text-default-regular: 400 0.875rem/1.25rem Inter;
+  --echoes-typography-text-default-medium: 500 0.875rem/1.25rem Inter;
+  --echoes-typography-text-default-semi-bold: 600 0.875rem/1.25rem Inter;
+  --echoes-typography-text-small-regular: 400 0.75rem/1.125rem Inter;
+  --echoes-typography-text-small-semi-bold: 600 0.75rem/1.125rem Inter;
+  --echoes-typography-text-large-regular: 400 1rem/1.5rem Inter;
+  --echoes-typography-text-large-semi-bold: 600 1rem/1.5rem Inter;
+  --echoes-typography-heading-xsmall: 700 0.75rem/1.125rem Inter;
+  --echoes-typography-heading-small: 700 0.875rem/1.25rem Inter;
+  --echoes-typography-heading-medium: 600 1rem/1.5rem Inter;
+  --echoes-typography-heading-large: 600 1.25rem/1.875rem Inter;
+  --echoes-typography-heading-xlarge: 600 1.5rem/2.25rem Inter;
+  --echoes-typography-code-default: 400 0.875rem/1.25rem 'Ubuntu Mono';
+  --echoes-typography-code-highlight: 700 0.875rem/1.25rem 'Ubuntu Mono';
+  --echoes-typography-code-comment: 400 italic 0.875rem/1.25rem 'Ubuntu Mono';
+  --echoes-typography-others-label: 600 0.875rem/1.25rem Inter;
+  --echoes-typography-others-helper-text: 400 0.75rem/1.125rem Inter;
+  --echoes-typography-display-default: 600 2.125rem/3.125rem Inter;
```

Comment:
@jeremy-davis-sonarsource do you want to merge it without updating all the usage of the typography tokens in our components ?